README.md

```diff
@@ -34,6 +39,31 @@ It is possible to send args to the script under test by adding -- after the load
 ./bin/load-runner.js -s ./scripts/dummy.js -c 5 -n 100 -r 10 -o -- -testarg1 testval1
 ```
 
+# Environment variables
+
+There are some environment variables generated for the test script that can be useful. You can use Node.js `process.env` object to access them.
+
+* `LR_FLOW_NUMBER` - number generated each run, value depends on percentages specified by `-f` parameter, see [Flow numbers](#flow-numbers)  
```

Comment:
One thing to keep in mind with env vars rather than parameters, that I've just thought of: for making changes to a script, and running it outside of load-runner, one would need to know to pass both the parameter values, and set env vars. Not necessarily a bad thing, just something to keep in mind. :)